docs/glossary.md

```diff
@@ -0,0 +1,194 @@
+
+### Data Availability Committee (DAC)
+A Data Availability Committee (DAC) is a trusted group of data managers (nodes) that store and grant access to data, maintaining the data off the main chain. Upon receiving and verifying data requests, the DAC provides authorized users with the needed data.
+
+### Data Attestation
+Data attestation is the process of verifying the authenticity and integrity of data. A block header on Avail contains two types of attestations. One attestation being KZG polynomial commitments of the submitted data and another being the root of the Merkle tree whose leaves are the data blobs. Supermajority of validators of Avail reach finality over the header by signing over a chain containing the header using the GRANDPA protocol.
+
+
+
+## Account and Asset Management
+
+### Avail JS Apps
+The Avail JS Apps UI is a forked version of [Polkadot JS Apps UI](https://polkadot.js.org/apps/) repository for visualizing and interacting with the Avail network.
+### Stash
+It is a private and secure storage for accounts. It is typically implemented as a separate wallet or account that is not connected to the main account or wallet. Stashes can be used to store a variety of assets, including cryptocurrencies, NFTs, and other digital assets.
+
+### Controller
+Controller is a person or entity that has the authority to manage an account. Controllers can perform a variety of tasks, such as: Transferring assets, Receiving assets, Setting permissions, Changing account settings, and Viewing account information.
+
+### AVL
+The native token of Avail.
+
+## Network Architecture
+### libp2p
+Libp2p is a modular network stack for building peer-to-peer (P2P) applications. It is open source and free to use, and it is used by Avail.
+### DHT (Distributed Hash Table)
+A distributed hash table (DHT) is a distributed system that provides a lookup service similar to a hash table. A DHT stores key-value pairs and allows peers to efficiently locate the value associated with a given key. The DHT plays a crucial role in the data cells (random sampling and proof verification) sharing process. It allows nodes to store and discover information about cell providers. Nodes in the network are interconnected via the DHT, enabling efficient cell discovery and retrieval.
+
+### Kademlia DHT (Kad-DHT)
+Kad-DHT is based on the idea of a chord ring, which is a logical ring of nodes that are ordered by their IDs. Each node in the ring is responsible for storing a subset of the data, and nodes can communicate with each other directly to retrieve data. The Kademlia Distributed Hash Table allows our nodes to store data cells and discover information about what peer is holding exactly what piece of data. Matrix data cells are mapped by unique Peer IDs.
```

Comment:
Should first just make it clear that "Kad-DHT is a type of DHT... based on..."

It's not clear how Avail uses Kad-DHT.